google-cloud-spanner/src/main/java/com/google/cloud/spanner/BackupInfo.java

```diff
@@ -18,6 +18,8 @@
 
 import com.google.api.client.util.Preconditions;
 import com.google.cloud.Timestamp;
+import com.google.spanner.admin.database.v1.EncryptionConfig;
```

Comment:
Shouldn't we be using the wrapper classes here instead of the generated classes?

Reply:
Oh thanks for pointing that out, I fixed it and created a wrapper class for the EncryptionInfo as well.